From: Markus Lehtonen <markus.lehtonen@linux.intel.com>
Date: Thu, 26 Feb 2015 11:12:42 +0200
Subject: [PATCH] gbp.conf: adapt to new patch-export option names

Signed-off-by: Markus Lehtonen <markus.lehtonen@linux.intel.com>
---
 .gbp.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.gbp.conf b/.gbp.conf
index 4af2c49..b31adf7 100644
--- a/.gbp.conf
+++ b/.gbp.conf
@@ -6,6 +6,6 @@ upstream-branch = srcdata/gbp-test2/upstream
 upstream-tag = srcdata/gbp-test2/upstream/%(upstreamversion)s
 packaging-dir = packaging
 patch-export = True
-patch-export-ignore-path = ^(packaging/.*|\.gbp.conf)
+patch-ignore-path = ^(packaging/.*|\.gbp.conf)
 spec-file = packaging/gbp-test2.spec
 
-- 
2.1.4

