From: Markus Lehtonen <markus.lehtonen@linux.intel.com>
Date: Thu, 12 Dec 2013 12:15:13 +0200
Subject: [PATCH] Update .gbp.conf

Signed-off-by: Markus Lehtonen <markus.lehtonen@linux.intel.com>
---
 .gbp.conf | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/.gbp.conf b/.gbp.conf
index 885f9bb..c611421 100644
--- a/.gbp.conf
+++ b/.gbp.conf
@@ -1,7 +1,7 @@
 [DEFAULT]
 vendor = Test-Distro
-packaging-branch = srcdata/gbp-test2/master
-packaging-tag = srcdata/gbp-test2/release/%(version)s
+packaging-branch = srcdata/gbp-test2/master-orphan
+packaging-tag = srcdata/gbp-test2/release-orphan/%(version)s
 upstream-branch = srcdata/gbp-test2/upstream
 upstream-tag = srcdata/gbp-test2/upstream/%(upstreamversion)s
 packaging-dir = packaging
-- 
2.1.4

